doc/manual/command-ref/conf-file.xml

```diff
@@ -864,6 +864,26 @@ requiredSystemFeatures = [ "kvm" ];
 
   </varlistentry>
 
+  <varlistentry xml:id="conf-tarball-ttl"><term><literal>tarball-ttl</literal></term>
+
+    <listitem>
+
+      <para>Default: <literal>3600</literal> seconds.</para>
+
+      <para>The number of seconds a download is cached in
+      <filename>$XDG_CACHE_HOME/nix/tarballs</filename>
+      are considered fresh.
+      </para>
+
+      <para>This is most relevant for fetching tarballs that change
+      remotely via <envar>NIX_PATH</envar> or <function>fetchGit</function>.
+      </para>
+
+      <para>Setting the TTL to <literal>0</literal> forces Nix to check
+      if the tarball is up to date by ETag if the remote server supports
+      it.</para>
```

Comment:
This suggests that `ETag` is only used if TTL=0, but it's always used if supplied by the server.

Reply:
This is complicated :)